src/core/eme/handle_session_events.ts

```diff
@@ -202,11 +204,11 @@ export default function handleSessionEvents(
         return (castToObservable(getLicense) as Observable<TypedArray|ArrayBuffer|null>)
           .pipe(
             timeout(10 * 1000),
-            catchError((error : Error) : never => {
+            catchError((error : Error|undefined) : never => {
               throw error instanceof TimeoutError ?
                 new EncryptedMediaError("KEY_LOAD_TIMEOUT",
                   "The license server took more than 10 seconds to respond.", false) :
-                error;
+                (error || new Error("Unknown error occured during license request."));
```

Comment:
I think `"Unknown"` here is unnecessary. It could even confuse the user which will not be able to link that with its `getLicense` callback